tcl/cast.tcl

```diff
@@ -374,3 +373,59 @@ proc qc::is_period {string} {
     }
 }
 
+namespace eval qc::cast {
```

Comment:
If the proc is defined as qc::cast::decimal are we able to centralise the creation of the ensemble and a list of what procs are exported ?